Workbooks/SapMonitor/SapHana/SapHana.workbook

```diff
@@ -2070,7 +2097,7 @@
+                                "query": "let param_sid = \"{param_sid}\"\r\n;\r\n// Filter the base data set to only work with the selected SID\r\nlet table_systemreplication = SapHana_SystemReplication_CL\r\n| where SYSTEM_ID_s == param_sid\r\n;\r\n// Find the latest post for this instance\r\nlet table_lastpost = table_systemreplication\r\n| summarize lastpost = max(TimeGenerated)\r\n| project lastpost\r\n;\r\n// Create table of the last posts for this instance\r\nlet table_hsrlastpost = table_systemreplication\r\n| join kind=inner (table_lastpost) \r\n  on  $left.TimeGenerated == $right.lastpost\r\n;\r\n//create a table with the primary site\r\nlet table_primarysite = table_hsrlastpost \r\n| distinct  \r\n    SITE_NAME_s\r\n| project  \r\n    NodeID = SITE_NAME_s,\r\n    SourceID = '',\r\n    TargetID = '',\r\n    CentreContent = SITE_NAME_s, \r\n    EdgeLabel = '', \r\n    TopContent = 'Primary'\r\n;\r\n// create a table with the secondary sites\r\nlet table_secondarysite = table_hsrlastpost \r\n| summarize\r\n   TIME_DIFF_SECONDS_d = avg(TIME_DIFF_SECONDS_d),\r\n   OPERATION_MODE_s = max(OPERATION_MODE_s),\r\n   REPLICATION_MODE_s = max(REPLICATION_MODE_s),\r\n   SYSTEM_REPLICATION_STATUS_s = max(SYSTEM_REPLICATION_STATUS_s)\r\n  by\r\n  SITE_NAME_s,\r\n   SECONDARY_SITE_NAME_s\r\n| project  \r\n   NodeID = SECONDARY_SITE_NAME_s,\r\n   SourceID = SITE_NAME_s,\r\n   TargetID = SECONDARY_SITE_NAME_s,\r\n   CentreContent = SECONDARY_SITE_NAME_s,\r\n   EdgeLabel = strcat( iif(SYSTEM_REPLICATION_STATUS_s == 'ACTIVE', 'ðŸŸ¢', 'ðŸŸ¥'), ' ', SYSTEM_REPLICATION_STATUS_s, ' - ', REPLICATION_MODE_s, ' with ',OPERATION_MODE_s), \r\n   TopContent = 'Secondary'\r\n;\r\n// join the two together\r\ntable_primarysite\r\n| union table_secondarysite\r\n| order by NodeID asc",
```

Comment:
not sure if this is an issue in the workbook, or github's diff tool or what.  in the before, i see:
![image](https://user-images.githubusercontent.com/10158007/97927976-e697df00-1d1a-11eb-9365-8a12452a67bd.png)
in the after i see:
`'ACTIVE', 'ðŸŸ¢', 'ðŸŸ¥')`

Reply:
Huh. I haven't touched that at all. Any suggestion on what I can do here?

Reply:
@persiaAziz use a text editor that is unicode capable so you don't use the emojis 🔥

Reply:
I just copied the template from the portal directly and pasted here. The template on the portal does not contain the green and red shapes .  What editor do you use?

Reply:
okay fixed  it